code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -965,7 +965,7 @@ Defined in conflicts.dm of the #defines folder.
 	w_class = 4
 	max_rounds = 3
 	current_rounds = 3
-	ammo = /datum/ammo/bullet/shotgun/buckshot/masterkey
+	ammo = /datum/ammo/bullet/shotgun/buckshot/masterkey, /datum/ammo/bullet/shotgun/flechette/masterkey
```

Comment:
`ammo` in this case is not a list of possible ammos, but a single kind. This will require a bit of a rework to work differently.

Reply:
Noticed, think I found a decent way to do it.